engines/director/detection_tables.h

```diff
@@ -5336,6 +5409,13 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1("mulanpresskit", "", "Disney's Mulan", "d16e0af2a86b6b04a8f9c86205c0d3d2", 1031738, 601),
 	WINGAME1("mulanpresskit", "", "MULAN.EXE", "ece1c7a090010965890840102978e693", 1828679, 601),
 
+	MACGAME2_l("muumitpiilosilla", "",
+			   "start", "1d9149b9745e35ef93ed4cdf08ec3af7", 1032405,
+			   "mum1dir/01DALEN.DXR", "c90f2bb4d9479eedc47b9742422448a4", 3679416, Common::FI_FIN, 650),
+	WINGAME2t_l("muumitpiilosilla", "",
+				"START.EXE", "f9522a8bce8bbc5300efa1699dee56c0", 1733522,
+				"MUM1DIR/01DALEN.DXR", "1e9170db3f9bac48df2ffd329b96f32d", 3679416, Common::FI_FIN, 606),
```

Comment:
Is there a 6.0.6? I don't think I've seen that version.

Reply:
Here's the version string I spotted in `START.EXE` in a hex editor:

<img width="176" alt="image" src="https://user-images.githubusercontent.com/780485/151718291-9f748bc7-66c1-4eb3-9bcb-bcd5998ebe40.png">

Reply:
See my other comment on this issue.

Reply:
Fixed.